pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -0,0 +1,355 @@
+              </div>
+              <div class="modal-body">
+                <form name="copy-assessment-group-form" method="POST" enctype="multipart/form-data">
+                  <div class="input-group mb-3">
+                    <select class="custom-select" name="inputGroupSelect01">
+                      <option selected>Choose Assessment...</option>
+                      <% assessment_list_rows.forEach(function(row, iRow) {%>
+                        <option value = <%= row.id %> ><%= row.tid %>: <%= row.title %></option>
+                      <% }); %>
+                    </select>
+                  </div>
+                  <div class="d-flex justify-content-end">
+                    <div class="form-group mb-0">
+                      <input type="hidden" name="__action" value="copy_assessment_groups">
+                      <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                      <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                      <button type="submit" class="btn btn-primary">Copy</button>
+                    </div>
+                  </div>
+                </form>
+              </div>
+            </div>
+          </div>
+        </div>
+      <div class="modal fade" id="addGroupModal" tabindex="-1" role="dialog" aria-labelledby="addGroup">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h4 class="modal-title" id="addGroupModalLabel">Add a group</h4>
+            </div>            
+              <form name="add-Group-form" method="POST">
+                <div class="modal-body">
+                  <div class="form-name">
+                    <label for="formName">Group Name</label>
+                    <input type="text" class="form-control" name="groupname">
+                  </div>
+                  <br>
+                  <div class="form-uids">
+                    <label for="formUids">UIDs (Seperate with ",". Please make sure they are not in any other groups)</label>
+                    <input type="text" class="form-control" name="uids" placeholder="a@illinois.edu, b@illinois.edu, c@illinois.edu">
+                  </div>
+                </div>    
+                <div class="modal-footer">
+                  <input type="hidden" name="__action" value="addGroup">
+                  <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">  
+                  <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                  <button type="submit" class="btn btn-primary">Add</button>
+                </div>
+              </form>
+          </div>
+        </div>
+      </div>
+
+      <div class="modal fade" id="deleteAllGroupsModal" tabindex="-1" role="dialog" aria-labelledby="deleteAllGroups">
```

Comment:
```suggestion
      <div class="modal fade" id="deleteAllGroupsModal" tabindex="-1" role="dialog" aria-labelledby="deleteAllGroupsModalLabel">
```
This should match the `id` on line 147. `aria-labelledby` tells screen readers where to find the title of this div.